test/functional/wallet_balance.py

```diff
@@ -64,8 +65,10 @@ def run_test(self):
         self.log.info("Test getbalance with different arguments")
         assert_equal(self.nodes[0].getbalance("*"), 50)
         assert_equal(self.nodes[0].getbalance("*", 1), 50)
-        assert_equal(self.nodes[0].getbalance("*", 1, True), 50)
+        assert_equal(self.nodes[0].getbalance("*", 1, True), 100)
         assert_equal(self.nodes[0].getbalance(minconf=1), 50)
+        assert_equal(self.nodes[0].getbalance(minconf=0, include_watchonly=True), 100)
```

Comment:
Just `include_watchonly=True`, if #15729 is the way to go? Same below.

Reply:
The two pulls will conflict either way, so I'd rather keep it